src/Microsoft.OData.Client/Materialization/EntryValueMaterializationPolicy.cs

```diff
@@ -671,6 +673,18 @@ private void MaterializeResolvedEntry(MaterializerEntry entry, bool includeLinks
             if (entity != null)
             {
                 entity.Context = this.EntityTrackingAdapter.Context;
+                // if NoTracking

```

Comment:
```suggestion

                // if NoTracking
```
Produces warning
```CSharp.Layout : A single-line comment must be preceded by a blank line or another single-line comment, or must be the first item in its scope. To ignore this error when commenting out a line of code, begin the comment with '////' rather than '//'.```